src/components/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -117,8 +117,6 @@ export const Interactive: StoryObj<Args> = {
     chromatic: {
       disableSnapshot: true,
     },
-    snapshot: {
-      skip: true,
-    },
+    snapshot: {},
```

Comment:
There's a comment to explain it but the story is for interaction purposes and since we can't snap multiple interactions, not much value snapping. Can enable tho, would be similar to a default snap

Reply:
ohh D'oh. i was (un-)commenting a bunch at once and forgot to review this one. I'll clean up this PR to not move as much around